sklearn/linear_model/_logistic.py

```diff
@@ -1422,15 +1463,19 @@ class LogisticRegressionCV(LogisticRegression, LinearClassifierMixin, BaseEstima
             - 'liblinear' might be slower in :class:`LogisticRegressionCV`
               because it does not handle warm-starting. 'liblinear' is
               limited to one-versus-rest schemes.
+            - 'newton-cholesky' is a good choice for n_samples >> n_features, but is
+              limited to one-versus-rest schemes.
```

Comment:
```suggestion
            - 'newton-cholesky' is a good choice for n_samples >> n_features,
              especially with one-hot encoded categorical feature with rare categories
              but is limited to binary classification and the one-versus-rest
              reduction for multiclass classification. Be aware that the memory usage
              of this solver has a quadratic dependency on `n_features` because it
              explicitly computes the Hessian matrix.
```

Reply:
I'll add this comment to the GLM too.